### ISSUE TITLE
Retinal gliosis

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#10869]
Preferred term label:
Retinal gliosis

Synonyms
	•	Glial proliferation in the retina
	•	Reactive gliosis of the retina

Definition (free text, please give PubMed ID)
A pathological condition characterized by the proliferation of glial cells in the retina, often occurring as a reactive response to injury, inflammation, or degenerative changes, and typically resulting in structural and functional alterations in the retina. 

Parent term (use hpo.jax.org/app)
[Retinal degeneration (HP:0000546)](https://hpo.jax.org/app/browse/term/HP:0000546)

Diseases characterized by this term? (e.g. Orphanet or OMIM number)
	•	Proliferative vitreoretinopathy

Your nano-attribution (ORCID)
[[0000-0001-9042-0508]](https://orcid.org/0000-0001-9042-0508)

**Comment by pnrobinson** [comment]
@daboe01 
Adding new term: 
- ID: HP:6001301
- label: Retinal gliosis
- parents: [Retinal degeneration (HP:0000546)](https://hpo.jax.org/app/browse/term/HP:0000546)
- pmid: PMID:26175331
- def: A pathological condition characterized by the proliferation of glial cells in the retina, often occurring as a reactive response to injury, inflammation, or degenerative changes, and typically resulting in structural and functional alterations in the retina.
- comment: 
- orcid: orcid:0000-0001-9042-0508
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10869
- synonyms:
